sql/item_basic.sql

```diff
@@ -3827,28 +3827,28 @@ INSERT INTO `item_basic` VALUES (4071,0,'copy_of_rems_tale,_chapter_8','rems_tal
 INSERT INTO `item_basic` VALUES (4078,0,'bolt_of_moonbow_cloth','moonbow_cloth',12,4,40,0,19250);
 INSERT INTO `item_basic` VALUES (4079,0,'square_of_moonbow_leather','moonbow_leather',12,4,41,0,19250);
 INSERT INTO `item_basic` VALUES (4080,0,'pot_of_moonbow_urushi','moonbow_urushi',12,4,43,0,19250);
 INSERT INTO `item_basic` VALUES (4081,0,'moonbow_stone','moonbow_stone',12,4,39,0,19250);
 INSERT INTO `item_basic` VALUES (4082,0,'piece_of_moonlight_coral','moonlight_coral',12,4,0,0,19250);
-INSERT INTO `item_basic` VALUES (4083,0,'handful_of_beryllium_arrowheads','ber._arrowheads',99,4,0,0,0);
-INSERT INTO `item_basic` VALUES (4084,0,'bag_of_mixed_fletchings','mixed_fletchings',99,4,0,0,0);
-INSERT INTO `item_basic` VALUES (4085,0,'handful_of_beryllium_bolt_heads','ber._bolt_heads',99,4,0,0,0);
-INSERT INTO `item_basic` VALUES (4086,0,'lustreless_scale','lustreless_scale',99,4,46,1,0);
+INSERT INTO `item_basic` VALUES (4083,0,'handful_of_beryllium_arrowheads','ber._arrowheads',99,4,0,0,134);
+INSERT INTO `item_basic` VALUES (4084,0,'bag_of_mixed_fletchings','mixed_fletchings',99,4,0,0,96);
+INSERT INTO `item_basic` VALUES (4085,0,'handful_of_beryllium_bolt_heads','ber._bolt_heads',99,4,0,0,235);
+INSERT INTO `item_basic` VALUES (4086,0,'lustreless_scale','lustreless_scale',99,4,46,1,20);
 INSERT INTO `item_basic` VALUES (4094,0,'bowl_of_goblin_stew_880','goblin_stew_880',1,61520,0,0,0);
 INSERT INTO `item_basic` VALUES (4095,0,'bismuth_sheet','bismuth_sheet',12,4,38,0,810);
-INSERT INTO `item_basic` VALUES (4096,0,'fire_crystal','fire_crystal',12,516,35,0,15);
-INSERT INTO `item_basic` VALUES (4097,0,'ice_crystal','ice_crystal',12,516,35,0,30);
-INSERT INTO `item_basic` VALUES (4098,0,'wind_crystal','wind_crystal',12,516,35,0,15);
-INSERT INTO `item_basic` VALUES (4099,0,'earth_crystal','earth_crystal',12,516,35,0,15);
-INSERT INTO `item_basic` VALUES (4100,0,'lightning_crystal','lightng._crystal',12,516,35,0,30);
-INSERT INTO `item_basic` VALUES (4101,0,'water_crystal','water_crystal',12,516,35,0,15);
-INSERT INTO `item_basic` VALUES (4102,0,'light_crystal','light_crystal',12,516,35,0,80);
-INSERT INTO `item_basic` VALUES (4103,0,'dark_crystal','dark_crystal',12,516,35,0,80);
+INSERT INTO `item_basic` VALUES (4096,0,'fire_crystal','fire_crystal',99,516,35,0,15);
+INSERT INTO `item_basic` VALUES (4097,0,'ice_crystal','ice_crystal',99,516,35,0,30);
+INSERT INTO `item_basic` VALUES (4098,0,'wind_crystal','wind_crystal',99,516,35,0,15);
+INSERT INTO `item_basic` VALUES (4099,0,'earth_crystal','earth_crystal',99,516,35,0,15);
+INSERT INTO `item_basic` VALUES (4100,0,'lightning_crystal','lightng._crystal',99,516,35,0,30);
+INSERT INTO `item_basic` VALUES (4101,0,'water_crystal','water_crystal',99,516,35,0,15);
+INSERT INTO `item_basic` VALUES (4102,0,'light_crystal','light_crystal',99,516,35,0,80);
+INSERT INTO `item_basic` VALUES (4103,0,'dark_crystal','dark_crystal',99,516,35,0,80);
```

Comment:
Hello, Delaide, thanks for your PR!

Crystals are still stackable by 12 on retail, not 99 (note from our discussion on Discord).

Reply:
@UynGH Okay, items have been set back to 12 as proper and confirmed on ffxi ah